maplibre-winit/src/input/zoom_handler.rs

```diff
@@ -24,18 +24,15 @@ impl UpdateState for ZoomHandler {
                 let view_proj = view_state.view_projection();
                 let inverted_view_proj = view_proj.invert();
 
-                if let Some(cursor_position) = view_state.camera().window_to_world_at_ground(
+                if let Some(cursor_position) = view_state.window_to_world_at_ground(
```

Comment:
try using slightly shorter names for vars when that var is used just for a few lines, and it is relatively easy to understand the meaning, e.g. `cursor_pos`?  I tend to use a single character vars for oneliners as those are the easiest

Reply:
I don't want to touch these functions right now because they contain heavy maths and the name makes sense from that code's perspective.